src/App/App.test.js

```diff
@@ -144,5 +145,252 @@ describe('App Component', () => {
     expect(movieCardIconFavorited).toBeInTheDocument();
   });
 
+  it('should add a new rating when a rating is submitted', async () => {
```

Comment:
Awesome job with all these integration tests! I think we're doing a great job getting all the user stories covered by our test suites